x/pairing/keeper/msg_server_stake_provider_test.go

```diff
@@ -722,3 +723,34 @@ func TestStakeEndpoints(t *testing.T) {
 		})
 	}
 }
+
```

Comment:
let's test that in a regular (correct) stake he is not frozen too,
let's add a test that he is frozen when staking a little when delegators add still frozen, but he can unfreeze and then unfrozen